.github/workflows/crowdin-i18n-curriculum-download.yml

```
@@ -1,6 +1,8 @@
 name: Crowdin Curriculum Download
 on:
   workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * *"
```

We should probably add a quick comment here explaining this runs every day at 12:00

```suggestion
    # runs everyday at 12:00 noon UTC
    - cron: "0 12 * * *"
```

here and elsewhere?

---

Good idea. 👍🏻